ansible/roles/gpu/files/Dockerfile.ubuntu

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu
+RUN apt update && apt install gpg curl -y
+RUN distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
+    && curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey |  apt-key add - \
+    && curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | tee /etc/apt/sources.list.d/nvidia-docker.list
+
+RUN  apt-get update &&  apt-get install -y nvidia-container-runtime
```

Comment:
Proof it needs this hook binary from the runtime

https://github.com/NVIDIA/nvidia-container-runtime/blob/77d22bf3605ba7e8903f723cd7cf2b4b07be1e17/cmd/nvidia-container-runtime/nvcr.go#L115